props/saber_fett263_G_buttons.h

```diff
@@ -0,0 +1,445 @@
+//
+// LOCKUP_THRESHOLD_G 2.0
+// This is the "delay" threshold to determine Clash vs Lockup
+//
+// 2 Button Controls (PWR and AUX):
+// "Battle Mode" - Swing while saber is OFF to ignite in Battle Mode
+// or hold AUX and Swing while blade is ON to toggle mode
+//
+// Ignite (ON) - click PWR while OFF or Swing Saber while OFF
+//
+// Muted Ignition (ON) - double click PWR while OFF
+// Retract (OFF) - click PWR while ON (not swinging or in Battle Mode)
+// Twist Off (OFF) if in Battle Mode
+// Play Music Track - hold and release PWR while OFF
+// Blast - click AUX while ON
+// Multi-Blast Mode - hold and release AUX while ON to enter mode, Swing to initiate Blasts, click Aux to exit mode
+// Clash - clash blade while ON
+// Stab - thrust forward and clash blade while ON
+// Lockup - hold AUX and clash while ON
+// Drag - hold AUX and clash pointing down while ON
+// Lightning Block - hold PWR and click AUX while ON
+// Melt - hold PWR (or AUX) and thrust forward and clash while ON
+// Force - hold and release PWR while ON
+// Color Change - hold AUX and click PWR while ON to enter CCWheel,
+// turn hilt to rotate through colors, click PWR to select/exit
+// if using COLOR_CHANGE_DIRECT each button press advances one Color at a time
+// Next Preset - click AUX while OFF
+// MULTI_PHASE Next Preset - hold AUX and TWIST while ON (use define to enable)
+// Previous Preset - hold AUX and click PWR while OFF
+// MULTI_PHASE Previous Preset - hold PWR and TWIST while ON (use define to enable)
+// Enter SA22C Volume Menu - hold and release AUX while OFF
+// Volume Up (10% increment, 100% max) - click PWR while in Volume Menu while OFF
+// Volume Down (10% increment) - click AUX while in Volume Menu while OFF
+// Exit Volume Menu - hold and release AUX while in Volume Menu while OFF
+
+#ifndef MOTION_TIMEOUT
+#define MOTION_TIMEOUT 60 * 15 * 1000
+#endif
+
+#ifndef LOCKUP_THRESHOLD_G
+#define LOCKUP_THRESHOLD_G 2.0
+#endif 
+
+#ifndef PROPS_SABER_FETT263_G_BUTTONS_H
+#define PROPS_SABER_FETT263_G_BUTTONS_H
+
+#include "prop_base.h"
+#include "../sound/hybrid_font.h"
+
+#undef PROP_TYPE
+#define PROP_TYPE SaberFett263Buttons
+
+// The Saber class implements the basic states and actions
+// for the saber.
+class SaberFett263Buttons : public PropBase {
+public:
+SaberFett263Buttons() : PropBase() {}
+  const char* name() override { return "SaberFett263Buttons"; }
+
+
+  // EVENT_SWING
+  bool swinging_ = false;
+  void Loop() override {
+    PropBase::Loop();
+    if(!SaberBase::IsOn() ) {
+      if (millis() - saber_off_time_ < MOTION_TIMEOUT) {
+        SaberBase::RequestMotion();
+        if (!swinging_ && fusor.swing_speed() > 250) {
+            swinging_ = true;
+            Event(BUTTON_NONE, EVENT_SWING);
+            }
+            if (swinging_ && fusor.swing_speed() < 100) {
+            swinging_ = false;
+            }
+      }
+    }
+    if (!swinging_ && fusor.swing_speed() > 250) {
```

Comment:
Why do this code twice?

Reply:
One is for EVENT_SWING while on, the second is for the SWING ON ignition, some users want to increase the speed needed to ignite the blade so having the version for !SaberBase::InOn() allows without changing the EVENT_SWING  for other uses while ON, like the Multi-Blast.

Reply:
then there needs to be comments in the code that clarifies that, otherwise it just looks redundant.